contrib/cirrus/runner.sh

```diff
@@ -46,10 +48,12 @@ _run_verify_vendor() {
 }
 
 _run_unit() {
+	cargo vendor
```

Comment:
Why do you want to run this  for unit and integration tests?

Reply:
it looks like there is some vendoring difference when comping with Fedora's rust toolchain vs upstream.  For whatever reason, cargo vendor puts things in order.

Reply:
i have one more idea to try ...  hang tight